app/src/main/java/com/pokerio/app/utils/GameState.kt

```diff
@@ -58,7 +61,7 @@ object GameState {
         // TODO: Load settings
```

Comment:
Could you take care of this TODO?

Reply:
I added it now, please check if it is ok.